scripts/run_unit_tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/../tests
+export PYTHONPATH=..
+python -m unittest discover
```

Comment:
Should we add a custom output path for xUnit XML report according to configuration in this other PR #4 ? 
 
      sonar.python.xunit.reportPath=target/surefire-reports/TEST-nosetests.xml